Library/Homebrew/cask/cask_loader.rb

```diff
@@ -384,7 +384,8 @@ def load(config:)
           end
 
           if (disable_date = json_cask[:disable_date].presence)
-            reason = DeprecateDisable.to_reason_string_or_symbol json_cask[:disable_reason], type: :cask
+            disable_reason = json_cask[:disable_reason] || json_cask[:deprecation_reason]
+            reason = DeprecateDisable.to_reason_string_or_symbol disable_reason, type: :cask
             disable! date: disable_date, because: reason
           end
```

Comment:
```suggestion
if (date = json_cask[:disable_date].presence)
            disable_reason = json_cask[:disable_reason] || json_cask[:deprecation_reason]
            because = DeprecateDisable.to_reason_string_or_symbol disable_reason, type: :cask
disable! date:, because:
end
```
possible refactor to clean things up a little